package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dash-ag-grid",
-    "version": "31.0.1",
+    "version": "31.2.0",
```

Comment:
Chatted with @alexcjohnson -- I was wrong with the version bump; we follow the _major_ version of Ag Grid only, so this would be a minor bump to `31.1.0`. Sorry for the confusion here...